test/e2e/driver/aut.py

```diff
@@ -117,6 +119,7 @@ def stop(self):
         LOG.info('Stopping AUT: %s', self.path)
         self.detach_context()
         local_system.kill_process(self.pid)
+        time.sleep(1)
```

Comment:
Please don't just add random `sleep()` calls without adding a comment explaining WHY it was added.

Someone will remove them in the future thinking they are some leftover debugging.

Reply:
Honestly it should be a `FIXME` saying something like:
```python
# FIXME: Implement waiting for process to actually exit.
```